website/docs/examples/multi_client_geoimage/README.md

```diff
@@ -0,0 +1,236 @@
+          "uri": {
+            "path": "/pvt",
+            "query": "service_name=gps"
+          },
+          "every_n": 1
+        }
+      ]
+    }
+  ]
+}
+```
+
+## 4. Run the client
+
+In a terminal, run the client:
+
+```bash
+python main.py --config config.json
+```
+
+you should see the following output:
+
+```bash
+Received event from gps/pvt
+Could not sync image and gps data
+Received event from oak0/rgb
+Received event from oak0/rgb
+Received event from oak0/rgb
+Received event from oak0/rgb
+Received event from oak0/rgb
+Received event from gps/pvt
+Skipping image because stamp_diff is too large: 0.4036272089870181
+Skipping image because stamp_diff is too large: 0.30224284599535167
+Skipping image because stamp_diff is too large: 0.19944817599025555
+Skipping image because stamp_diff is too large: 0.09692963500856422
+Synced image and gps data with stamp_diff: 0.006475306989159435
+Received event from oak0/rgb
+Received event from oak0/rgb
+Received event from oak0/rgb
+Received event from oak0/rgb
+Received event from gps/pvt
+Skipping image because stamp_diff is too large: 0.32480611599748954
+Skipping image because stamp_diff is too large: 0.22475609098910354
+Skipping image because stamp_diff is too large: 0.12418683301075362
+Synced image and gps data with stamp_diff: 0.024251561000710353
+Received event from oak0/rgb
+Received event from oak0/rgb
+Received event from oak0/rgb
+Received event from gps/pvt
+Skipping image because stamp_diff is too large: 0.23738648000289686
+Skipping image because stamp_diff is too large: 0.09838183000101708
+Synced image and gps data with stamp_diff: 0.039805378997698426
+Received event from oak0/rgb
+Received event from oak0/rgb
+Received event from oak0/rgb
+Received event from oak0/rgb
+Received event from gps/pvt
+Skipping image because stamp_diff is too large: 0.3453739270044025
+Skipping image because stamp_diff is too large: 0.24114773501059972
+Skipping image because stamp_diff is too large: 0.14996879699174315
+Skipping image because stamp_diff is too large: 0.05010190900065936
+Could not sync image and gps data
+Received event from oak0/rgb
+Received event from oak0/rgb
+Received event from oak0/rgb
+Received event from oak0/rgb
+Received event from gps/pvt
+Skipping image because stamp_diff is too large: 0.36291974800406024
+Skipping image because stamp_diff is too large: 0.2664781129860785
+Skipping image because stamp_diff is too large: 0.1661627120047342
+Skipping image because stamp_diff is too large: 0.06768680198001675
+Could not sync image and gps data
+Received event from oak0/rgb
+Received event from oak0/rgb
+Received event from oak0/rgb
+Received event from oak0/rgb
+Received event from gps/pvt
+Skipping image because stamp_diff is too large: 0.38380739899002947
+Skipping image because stamp_diff is too large: 0.28419670299626887
+Skipping image because stamp_diff is too large: 0.1739979569974821
+Skipping image because stamp_diff is too large: 0.07935888497740962
```

Comment:
Maybe we shouldn't add so many prints in the website example?

Reply:
Also, not sure if this is too much work, but it could be nice to display the geotagged image along the GPS coordinates. Something similar to the [camera_client example](https://amiga.farm-ng.com/docs/examples/camera_client/)